src/views/helpers/getFormGroupClass.js

```diff
@@ -0,0 +1,12 @@
+// getFormGroupClass Helper
```

Comment:
Now you've done the refactoring to simplify how we set the class for form groups in the html, what this method is actually doing is determining whether we also need to apply `form-group-error`.

So how would you feel about renaming it `getFormGroupErrorClass`?

Reply:
Yes, agreed, I will make this change.